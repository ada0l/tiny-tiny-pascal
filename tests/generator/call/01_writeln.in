begin
writeln('hello, world', 'hello', 123);
writeln(3*(-3), 'hello', 123);
end.