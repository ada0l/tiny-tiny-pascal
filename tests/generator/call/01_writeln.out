hello, worldhello123
-9hello123
